examples/all-clusters-app/nrfconnect/README.md

```diff
@@ -0,0 +1,365 @@
+    the nRF Connect SDK after initializing environment variables.
+
+3.  Initialize environment variables referred to by the CHIP and the nRF Connect
+    SDK build scripts. Replace _nrfconnect-dir_ with the path to your nRF
+    Connect SDK installation directory, and _toolchain-dir_ with the path to GNU
+    Arm Embedded Toolchain.
+
+         $ source nrfconnect-dir/zephyr/zephyr-env.sh
+         $ export ZEPHYR_TOOLCHAIN_VARIANT=gnuarmemb
+         $ export GNUARMEMB_TOOLCHAIN_PATH=toolchain-dir
+
+4.  Update the nRF Connect SDK to the most recent supported revision by running
+    the following command (replace _matter-dir_ with the path to Matter
+    repository directory):
+
+         $ cd matter-dir
+         $ python3 scripts/setup/nrfconnect/update_ncs.py --update
+
+Now you can proceed with the [Building](#building) instruction.
+
+<hr>
+
+<a name="building"></a>
+
+## Building
+
+Complete the following steps, regardless of the method used for setting up the
+environment:
+
+1.  Navigate to the example's directory:
+
+        $ cd examples/all-clusters-app/nrfconnect
+
+2.  Run the following command to build the example, with _build-target_ replaced
+    with the build target name of the Nordic Semiconductor's kit you own, for
+    example `nrf52840dk_nrf52840`:
+
+         $ west build -b build-target
+
+    You only need to specify the build target on the first build. See
+    [Requirements](#requirements) for the build target names of compatible kits.
+
+The output `zephyr.hex` file will be available in the `build/zephyr/` directory.
+
+### Removing build artifacts
+
+If you're planning to build the example for a different kit or make changes to
+the configuration, remove all build artifacts before building. To do so, use the
+following command:
+
+    $ rm -r build
+
+### Building with release configuration
+
+To build the example with release configuration that disables the diagnostic
+features like logs and command-line interface, run the following command:
+
+    $ west build -b build-target -- -DOVERLAY_CONFIG=third_party/connectedhomeip/config/nrfconnect/app/release.conf
+
+Remember to replace _build-target_ with the build target name of the Nordic
+Semiconductor's kit you own.
+
+### Building with low-power configuration
+
+You can build the example using the low-power configuration, which enables
+Thread's Sleepy End Device mode and disables debug features, such as the UART
+console or the **LED 1** usage.
+
+To build for the low-power configuration, run the following command with
+_build-target_ replaced with the build target name of the Nordic Semiconductor's
+kit you own (for example `nrf52840dk_nrf52840`):
+
+    $ west build -b build-target -- -DOVERLAY_CONFIG=overlay-low_power.conf
+
+For example, use the following command for `nrf52840dk_nrf52840`:
+
+    $ west build -b nrf52840dk_nrf52840 -- -DOVERLAY_CONFIG=overlay-low_power.conf
+
+#### Changing flash memory settings
```

Comment:
I would remove this section, as it is related strictly to MCUBoot, which is not available in this example.